radio_beam/beam.py

```diff
@@ -429,9 +429,29 @@ def deconvolve(self, other, failure_returns_pointlike=False):
                     pa=new_pa)
 
     def __eq__(self, other):
-        if ((self.major == other.major) and
-            (self.minor == other.minor) and
-            (self.pa == other.pa)):
+
+        # Catch floating point issues
+        atol_deg = 1e-12 * u.deg
+
+        # PA direction can be ambiguous so check both cases
+        zero_deg = 0.0 * u.deg
+
+        this_pa = self.pa.to(u.deg)
+        other_pa = other.pa.to(u.deg)
+
+        if np.abs(this_pa - other_pa) < atol_deg:
+            equal_pa = True
+        elif other_pa < zero_deg and this_pa > zero_deg:
+            equal_pa = np.abs(this_pa - (other_pa + 180 * u.deg)) < atol_deg
```

Comment:
It would probably be better to do:
```
this_pa = this_pa % (180*u.deg)
other_pa = other_pa % (180*u.deg)
```
instead of this `if` block, since both could be any arbitrary angle (don't know why, but someone could give 480 instead of 120)